src/components/Endpoint.tsx

```diff
@@ -258,35 +372,12 @@ const Endpoint = (props) => {
           </div>
 
           <div id='transfer-direction'>
-            <div className='form-check form-check-inline mt-4'>
-              <input
-                className='form-check-input'
-                onChange={handleTransferDirection}
-                type='radio'
-                name='transfer-direction'
-                id='transfer-to-jupyter'
-                value='transfer-to-jupyter'
-              />
-              <label className='form-check-label' htmlFor='transfer-to-jupyter'>
-                Transfer to Jupyterlab
-              </label>
-            </div>
-            <div className='form-check form-check-inline'>
-              <input
-                className='form-check-input'
-                onChange={handleTransferDirection}
-                type='radio'
-                name='transfer-direction'
-                id='transfer-from-jupyter'
-                value='transfer-from-jupyter'
-              />
-              <label className='form-check-label' htmlFor='transfer-from-jupyter'>
-                Transfer from Jupyterlab
-              </label>
-            </div>
-            <div className='form-check form-check-inline pl-0'>
-              <button className='btn btn-sm btn-primary' onClick={handleTransferRequest} type='button'>
-                Submit Transfer Request
+            <div className='btn-group mt-4'>
+              <button type='button' className='btn btn-sm btn-outline-secondary' onClick={handleTransferToJupyter}>
+                <i className='fa-solid fa-arrow-left'></i> Transfer To JupyterLab
+              </button>
+              <button type='button' className='btn btn-sm btn-outline-secondary' onClick={handleTransferFromJupyter}>
+                <i className='fa-solid fa-arrow-right'></i> Transfer From JupyterLab
```

Comment:
These look much better! Personally for me, the arrows make the direction of the transfer much more intuitive, whether files are being transferred from or into JupyterLab